packages/kbn-es-archiver/src/cli.ts

```diff
@@ -202,12 +201,22 @@ export function runCli() {
           WARNING: If the indices exist already they will be deleted!
 
           $ node scripts/es_archiver load my_test_data --config ../config.js
+
+        Example with Performance Option Override:
+          Load the [my_test_data] snapshot and override the default for
+          the batch size, and the default for the concurrency.
+
+          $ node scripts/es_archiver load my_test_data --config ../config.js --batch-size 300 --concurrency 2
```

Comment:
Suggest to shorten this a bit, e.g. something like this:
```suggestion
          The same with overrides for batch size and concurrency default values:

          $ node scripts/es_archiver load my_test_data --config ../config.js --batch-size 300 --concurrency 2
```
What do you think?